### ISSUE TITLE
Spot Pricing for EC2 instances in Asia Pacific region is not available

### ISSUE THREAD

**Issue by prajjwaldimri** [infracost/cloud-pricing-api#258]
The query below returns empty prices arrays.

```graphql
query {
  products(
    filter: {
      vendorName: "aws"
      region: "ap-south-1"
      service: "AmazonEC2"
      attributeFilters: [
        { key: "instanceType", value: "m5d.large" }
      ]
    }
  ) {
    prices(filter: { purchaseOption: "spot" }) {
      USD
      purchaseOption
      termPurchaseOption
    }
  }
}
```

Prices for all other regions are available in the cloud pricing api. AWS also has information about prices for these regions.

**Comment by aliscott** [comment]
Thanks for reporting @prajjwaldimri 🙏. We'll try and get a fix out for this soon.

**Comment by aliscott** [comment]
@prajjwaldimri this should be fixed now in v0.3.8. We've pushed this fix out to our hosted API as well.

**Comment by prajjwaldimri** [comment]
@aliscott Thanks for the quick fix. It's working now.
